### DISCUSSION TITLE
Nested elements fetch [Nov 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4453] [Discord Threads]
> 🙋‍♂️ **lublak**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### you can pass fields and limit query parameter for nested child list...  
#  
> 👨‍💻 **pranavxc**  
#### sample    
#### ```   
#### nested[Elements][fields]=Id,Title,Title2&nested[Elements][limit]=100   
#### ```  
#  
> 🙋‍♂️ **lublak**  
#### so as a get parameter?  
#  
> 🙋‍♂️ **lublak**  
#### can i use `*`  
#  
> 👨‍💻 **pranavxc**  
#### Yes you need to pass it as get url query params  
#  
> 👨‍💻 **pranavxc**  
#### and passing `*` will return all fields value  
#  
> 👨‍💻 **pranavxc**  
#### `nested[Elements][fields]=*&nested[Elements][limit]=100`  
#  
> 👨‍💻 **pranavxc**  
#### Sample url ```   
#### http://localhost:8080/api/v1/db/data/noco/p_86i5703ceziffc/md_l6bc1rxuzmsdfr/views/vw_lghk2baf6etznd?offset=0&limit=25&nested[Actors][fields]=*&nested[Elements][limit]=100   
#### ```  
#  
> 🙋‍♂️ **lublak**  
#### thanks i will try it out on monday  
#  
> 🙋‍♂️ **lublak**  
#### <@682854952657616902> hi! It works fine :) Is there also a possibility to limit the columns? So about permissions?   
#### E.g. I can create a new view at parent and hide columns. These are then no longer included in the rest api and can never be retrieved. Also I can create e.g. a filter.   
#### The possibility to create a view for the child would be relatively important.  
#  
> 👨‍💻 **pranavxc**  
#### <@100849859006840832> at the moment it's not possible, you can raise a feature request in our Github 👍  
#  
> 🙋‍♂️ **lublak**  
#### Oki!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1038164443420430377)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
